diff --git a/crates/trueos-v/src/qjs_abi.rs b/crates/trueos-v/src/qjs_abi.rs
index d3ce1afbf..8bc46f439 100644
--- a/crates/trueos-v/src/qjs_abi.rs
+++ b/crates/trueos-v/src/qjs_abi.rs
@@ -7,11 +7,5 @@ pub use crate::bp_abi::*;
 pub use crate::legacy_fs_abi::*;
 
 unsafe extern "C" {
-    pub fn trueos_cabi_gfx_texture_dimensions(
-        tex_id: u32,
-        out_width: *mut u32,
-        out_height: *mut u32,
-    ) -> i32;
-    pub fn trueos_cabi_gfx_texture_status(tex_id: u32) -> i32;
     pub fn trueos_cabi_boot_timestamp_secs() -> u64;
 }

